examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -11288,7 +11288,7 @@
               "singleton": 0,
               "bounded": 0,
               "defaultValue": "0x103",
-              "reportable": 0,
+              "reportable": 1,
```

Comment:
Fwiw, Matter now ignores this value in the .zap file, so there's not much reason to keep changing it anymore.

Reply:
Yes, "reportable" field should stay as 1 for now.